Description: Install the dma binary with the correct permissions.
 As per the Debian Policy, change the /usr/sbin/dma permissions from 2555
 to 2755, since removing root write access gains us nothing.
 Also, install the etc/ files with the correct owner and group; it seems
 that Debian pmake's bsd.files.mk does not understand SHAREOWN and SHAREGRP.
Origin: other: http://svn.ringlet.net/svn/ringlet/mail/dma/
Forwarded: not-needed
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2010-06-02

--- a/Makefile
+++ b/Makefile
@@ -15,7 +15,7 @@
 
 BINOWN= root
 BINGRP= mail
-BINMODE=2555
+BINMODE=2755
 WARNS?=	6
 
 # The pmake Debian package does not seem to deal properly with y.tab.h
--- a/etc/Makefile
+++ b/etc/Makefile
@@ -1,8 +1,8 @@
 # $DragonFly: src/etc/dma/Makefile,v 1.3 2008/02/12 22:10:20 matthias Exp $
 
 FILESDIR=	/etc/dma
-SHAREOWN=	root
-SHAREGRP=	mail
+FILESOWN=	root
+FILESGRP=	mail
 FILESMODE=	640
 
 .if !exists(${DESTDIR}/etc/dma/auth.conf)
